model-optimizer/extensions/front/onnx/pad_ext.py

```diff
@@ -28,11 +29,13 @@ class PadFrontExtractor(FrontExtractorOp):
     @classmethod
     def extract(cls, node):
         mode = onnx_attr(node, 'mode', 's', default='constant', dst_type=lambda x: x.decode())
-        if get_onnx_opset_version(node) < 11:
-            pads = onnx_attr(node, 'pads', 'ints', dst_type=lambda x: np.array(x, dtype=np.int64))
+        # Pytorch 1.3 while converting to opset 11, creates Pad from older opset.
+        # To be able to convert such models we have to check if pads attribute exists.
+        pads = onnx_attr(node, 'pads', 'ints', dst_type=int64_array)
```

Comment:
What is about [Pad-1](https://github.com/onnx/onnx/blob/master/docs/Changelog.md#Pad-1) that has `paddings` attribute?

Reply:
opset-1 is out of our scope, was never supported.